.github/workflows/workflow.yaml

```diff
@@ -107,6 +107,205 @@ jobs:
       - name: Comment coverage
         uses: coroo/pytest-coverage-commentator@v1.0.2
 
+  format:
```

Comment:
Random location: I just noticed we do not have an integration test for the binary file format (we do for csv and images). We should add an integration test for binary file format to see it works e2e as well